libraries/AP_Notify/RGBLed.h

```diff
@@ -105,12 +105,12 @@ class RGBLed: public NotifyDevice {
     const uint32_t sequence_disarmed_bad_gps = DEFINE_COLOUR_SEQUENCE_SLOW(BLUE);
 
     uint8_t last_step;
-    enum rgb_source_t {
+    enum class rgbSource {
```

Comment:
```suggestion
    enum class Source {
```

Reply:
@peterbarker san.
I agreed and made the change.